lib/utility/logger.rb

```diff
@@ -20,12 +21,16 @@ def level=(log_level)
       end
 
       def logger
-        @logger ||= ::Logger.new(STDOUT)
+        @logger ||= Settings[:ecs_logging] ? EcsLogging::Logger.new(STDOUT) : ::Logger.new(STDOUT)
       end
 
       SUPPORTED_LOG_LEVELS.each do |level|
         define_method(level) do |message|
-          logger.public_send(level, message)
+          if logger.is_a?(EcsLogging::Logger)
+            logger.public_send(level, message, service: { name: 'connectors-ruby' })
```

Comment:
Hi @tarekziade , according to [ECS documentation](https://www.elastic.co/guide/en/ecs/8.4/ecs-custom-fields-in-ecs.html), custom fields should be added under `labels`, but we should try to utilize the defined fields first.

I found that [service.name](https://www.elastic.co/guide/en/ecs/8.4/ecs-service.html#field-service-name) is the most appropriate field for `app_source`:

```
{"@timestamp":"2022-09-15T03:14:56.569Z","log.level":"INFO","message":"Successfully connected to Elasticsearch","ecs.version":"1.4.0","service":{"name":"connectors-ruby"}}
```